tox.ini

```diff
@@ -32,7 +33,7 @@ allowlist_externals =
     sh
 
 [testenv:linters]
-basepython = python3.10
```

Comment:
watch out changing the base python here (linters only).  Although not a problem in wazo-chatd because mypy is not used/enabled (yet!), any annotations using the pipe `|` symbol for type unions (which we allow because of lazy evaluations aka `from __future__ import annotations`) will trigger a `TypeError: unsupported operand type(s) for |: 'type' and 'type')` when checked with a version below 3.10

ref: https://wazo-platform.org/uc-doc/contributors/typing#lazy-annotations

Reply:
Same comment as Seb, with more background info

Reply:
> Although not a problem in wazo-chatd because mypy is not used/enabled (yet!)

Good point, if any typing is added in this PR, we must enable mypy too

Reply:
Since adding mypy raise a metric ton of errors from others parts of the code (`Found 62 errors in 34 files (checked 123 source files)`).
I'll just remove the type checking from this PR instead of modifying code unrelated to the ticket. Created [WAZO-3802](https://wazo-dev.atlassian.net/browse/WAZO-3802) for that.